### PR TITLE
Gameini database update.

### PR DESCRIPTION
**Pull request by Linktothepast**
Includes gameinis for the Pal and Ntsc region of TurboGrafx-16, some
Commodore 64 gameinis, removes old notes about mmu games, enforces
virtual xfb in games fixed by PR #1818, updates Star Wars: Rogue Leader
and Star Wars - Rogue Squadron III - Rebel Strike
according to the zfreeze pr updates,  MySims Kingdom is added and
Sakura Wars: So Long, My Love is updated.
Fixes issues 8091 and 8086.

### CODE REVIEW

**Comment by Linktothepast** [comment]
Also some gameinis that have dolphin capitalized were accidentally included while trying not to have conflicts with pr 1890, please ignore those.

File: Data/Sys/GameSettings/GLRD64.ini [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 [Core]
-# Values set here will override the main dolphin settings.
+# Values set here will override the main Dolphin settings.
 MMU = 1
 
 [EmuState]
 # The Emulation State. 1 is worst, 5 is best, 0 is not set.
-EmulationIssues =
+EmulationIssues = Needs lle audio for proper sound.
```
Comment (magumagu) [inline Data/Sys/GameSettings/GLRD64.ini]: Capitalize "LLE"?

**Comment by Linktothepast** [comment]
Made a new commit.
"Capitalizes LLE audio and changes star wars rogue leader and rebel
strike emulation state, addition of Rabbids Go Home and Bad Boys II in
the database, and enables mmu in Need for speed most wanted (gc) to
avoid crashes during loading a stage."

File: Data/Sys/GameSettings/010E01.ini [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# 010E01 - Wii Backup Disc v1.31
+
+[Core]
+# Values set here will override the main Dolphin settings.
+
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId =
```
Comment (Buddybenj) [inline Data/Sys/GameSettings/010E01.ini]: This should be set to 0.

**Comment by Linktothepast** [comment]
Made a new commit:
"Removes speedhacks from metroid prime 1 and 2, and fire emblem series
and update old notes for metroid prime. Capitalize dolphin in the few
inis that weren't from the previous commit."

File: Data/Sys/GameSettings/010E01.ini (hunk 2 above)
Comment (Linktothepast) [inline Data/Sys/GameSettings/010E01.ini]: The result is the same as undefined (not set) with 0.

Comment (Buddybenj) [inline Data/Sys/GameSettings/010E01.ini]: OK. I think we should be consistent, so do the rest of the INIs typically use nothing or 0?

Comment (Linktothepast) [inline Data/Sys/GameSettings/010E01.ini]: No ini has 0 value, they either have nothing or 1-5 values.

**Comment by Linktothepast** [comment]
Pushed another commit:
"Add Hyrule Field Speed Hack v2 by TheWearyGamer for the wii us version
of Zelda Twilight Princess. Enable mmu and disable anti-aliasing in
Fatal Frame IV: Mask of the Lunar Eclipse fixing issues 8108 and 8109."

**Comment by CarlKenner** [comment]
What is the speedhack for metroid prime and why did you remove it?

**Comment by Linktothepast** [comment]
I have not much clue what they are, they are old and i didn't find them much useful nowdays. If i remember correctly they messed with idleskipping, i removed them in case they cause some issue, any benefit they might cause is trivial.

**Comment by Linktothepast** [comment]
Pushed a new commit:
"Remove safetexturecache setting from Zelda Twilight Princess and Star
Wars Rogue Squadron/Rebel Strike based on the pr #1916 changes."

**Comment by Daniel130698** [comment]
Hi. I'm having problems with playing zelda Twilight Princess on dolphin. I'm using the gamecube version. My problem is that when I'm opening my map, the emulator crashes! Anyone who could help me?

**Comment by phire** [comment]
This is not really the right place for that.

Try our forums https://forums.dolphin-emu.org/Forum-support

**Comment by Daniel130698** [comment]
ok. thx

**Pull request merged**